inspirehep/modules/references/view_utils.py

```diff
@@ -85,7 +82,8 @@ def references(self):
                 else:
                     # Easier to use record macros over reference records.
                     if 'publication_info' in reference:
-                        reference['publication_info'] = [reference['publication_info']]
+                        reference['publication_info'] = [
+                            reference['publication_info']]
                     else:
                         reference['publication_info'] = []
```

Comment:
This looks like a `force_force_list`, but I swear I remember already making this comment on this piece of code...